pkg/kubelet/cm/memorymanager/memory_manager.go

```diff
@@ -82,6 +81,9 @@ type Manager interface {
 	// and is consulted to achieve NUMA aware resource alignment among this
 	// and other resource controllers.
 	GetPodTopologyHints(*v1.Pod) map[string][]topologymanager.TopologyHint
+
+	// GetMemory provides NUMA nodes that used to allocate the container memory
```

Comment:
nit: `s/nodes that used/nodes that are used/`

Reply:
done